### PR TITLE
Added websocket cookbook entry

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR adds information about how to configure and run websockets.

### CODE REVIEW

**Comment by wachterjohannes** [comment]
@chirimoya could you please post an example how to configure supervisor to run the websocket?

File: cookbook/websocket.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+-------------
+
+If you use the config layout of `sulu-standard
+<https://github.com/sulu-io/sulu-standard>`_ you only have to set following
+parameters.
+
+.. list-table::
+
+    * - websocket_url
+      - One url where the websocket is reachable.
```
Comment (danrot) [inline cookbook/websocket.rst]: Why one URL? Are there multiple ones?

File: cookbook/websocket.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    * - websocket_port
+      - The port where thewebsocket should listen on.
```
Comment (danrot) [inline cookbook/websocket.rst]: `The port on which the websocket should listen`

File: cookbook/websocket.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+can only be displayed when the websocket is running.
+
+Configuration
+-------------
+
+If you use the config layout of `sulu-standard
+<https://github.com/sulu-io/sulu-standard>`_ you only have to set following
+parameters.
+
+.. list-table::
```
Comment (danrot) [inline cookbook/websocket.rst]: Where are these configurations? Is not clear to me. Are you talking about the `parameters.yml` file?

File: cookbook/websocket.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+    sulu_websocket:
+        server:
+            http_host: %websocket_url%
+            port: %websocket_port%
+
+.. warning::
+
+    Because of a bug in the current sulu version you also have to set the
+    ``app/config/admin/config.yml`` ``sulu_content: preview: websocket: true``.
```
Comment (danrot) [inline cookbook/websocket.rst]: That's not a common description... I would just write that you have to set the value of `sulu_content.preview.websocket` to `true`. It also doesn't really matter in which file this is done.

File: cookbook/websocket.rst [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+Execution
+---------
+
+To execute the websocket you have to run the command
+``app/console sulu:websocket:run -e prod``. This command starts the server and
+listen to the configured port.
```
Comment (danrot) [inline cookbook/websocket.rst]: listens

File: cookbook/websocket.rst [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+    * - websocket_url
+      - One url where the websocket is reachable.
+    * - websocket_port
+      - The port where thewebsocket should listen on.
+
+If you have multiple installations of sulu on a single server be sure that you
+use unique ports per installation.
+
+if you have you own application you can configure the url and port in the
```
Comment (danrot) [inline cookbook/websocket.rst]: `If you have your own application, you can configure...`

File: cookbook/websocket.rst (hunk 1 above)
Comment (wachterjohannes) [inline cookbook/websocket.rst]: there could be ... if you have multiple domains configured to "find" sulu you could add here one of them. but that might be confusing. i will remove that.

**Comment by wachterjohannes** [comment]
@danrot i have fixed the typo / writing issues. but i will wait for @chirimoya to add the supervisord config

**Comment by wachterjohannes** [comment]
@danrot i have added now the example config.

**Pull request merged**